uvahelpr_web/marketplace/templates/marketplace/jobslist.html

```diff
@@ -0,0 +1,11 @@
+{% extends "marketplace/base.html" %}
+{% block content %}
+<h1>View Job Postings</h1>
+{% for job in job_list %}
```

Comment:
Use bootstrap's [grid system](http://getbootstrap.com/css/#grid)

Make sure your content is nested inside row's and columns so that the entire page is responsive to different devices.